Menu/Menu.js

```diff
@@ -1,10 +1,14 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
+    // Toggle the "menu--open" class on your menu refence.
+    menu.classList.toggle("menu--open");
+};
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector('.menu-button');
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener("click", () => {
+    toggleMenu();
```

Comment:
You could lose this wrapping function and just use `toggleMenu` as your callback:
```javascript
menuButton.addEventListener("click", toggleMenu);
```